install/self_managed/01_guided_setup.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Step 1: Guided Setup'
+install_track_title: Installation Guide (Self-Managed)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+import CollectorPkgInstallInstructions from '../../components/CollectorPkgInstallInstructions'
+
+To begin guided setup, you'll need to install the collector, which sends statistics information
+and query activity to pganalyze.
+
+First, you need to know which version of Ubuntu/Debian you are running. You can find out with this command:
+
+```
+lsb_release -a
+```
+
+Note that guided setup is supported for Ubuntu 14.04 or newer or Debian 10 or newer. If you are
+running an older version, please follow the manual setup instructions.
+
+If your OS distribution is supported, run the appropriate commands for your version:
+
+<CollectorPkgInstallInstructions kind='deb' />
```

Comment:
I suspect this should be a follow-up PR, but we should add Ubuntu 20.04 to this.

Reply:
Yeah, let's do those separately. I'll start an issue.